src/Sylius/Bundle/AdminBundle/Provider/StatisticsDataProviderInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Provider;
+
+use Sylius\Component\Core\Model\ChannelInterface;
+
+interface StatisticsDataProviderInterface
+{
+    public function getRawData(ChannelInterface $channel, \DateTimeInterface $startDate, \DateTimeInterface $endDate, string $interval): array;
```

Comment:
`array` return type on an interface makes it almost impossible to maintain BC, we should define its structure via a docblock or VO.

Reply:
I believe, the problem occurs, because we are doing a mapping from VO to array due to the serialization problem. Removing this constraint and usage of JMS/Symfony Serializer should resolve the problem